pkg/function/message.go

```diff
@@ -9,22 +9,32 @@ var (
 
 // Message is used to wrap the data return by UDF functions
 type Message struct {
-	key   string
+	key   []string
 	value []byte
 }
 
+// Key returns message key
+func (m *Message) Key() []string {
+	return m.key
+}
+
+// Value returns message value
+func (m *Message) Value() []byte {
+	return m.value
+}
+
 // MessageToDrop creates a Message to be dropped
 func MessageToDrop() Message {
-	return Message{key: DROP, value: []byte{}}
+	return Message{key: []string{DROP}, value: []byte{}}
 }
 
 // MessageToAll creates a Message that will forward to all
 func MessageToAll(value []byte) Message {
-	return Message{key: ALL, value: value}
+	return Message{key: []string{ALL}, value: value}
```

Comment:
Technically, `MessageToAll` should also be able to give keys, right?

Reply:
yeah once we have tags